File: docs/examples/quickstart/python/expense.py
from dataclasses import dataclass


@dataclass
class Expense:
    amount: int
    description: str
    submitted_by: str


EXPENSES = {
    1: Expense(500, "coffee", "alice@example.com"),
    2: Expense(5000, "software", "alice@example.com"),
    3: Expense(50000, "flight", "bhavik@example.com"),
}

File: docs/examples/quickstart/nodejs/expense.js
class Expense {
  constructor(amount, description, submittedBy) {
    this.amount = amount;
    this.description = description;
    this.submittedBy = submittedBy;
  }
}

const EXPENSES = {
  1: new Expense(500, 'coffee', 'alice@example.com'),
  2: new Expense(5000, 'software', 'alice@example.com'),
  3: new Expense(50000, 'flight', 'bhavik@example.com'),
};

module.exports = { Expense, EXPENSES };
